model/riscv_insts_mext.sail

```diff
@@ -11,27 +11,26 @@
 
 /* ****************************************************************** */
 
-union clause ast = MUL : (regidx, regidx, regidx, bool, bool, bool)
+union clause ast = MUL : (regidx, regidx, regidx, mul_op)
 
-mapping encdec_mul_op : (bool, bool, bool) <-> bits(3) = {
-  (false, true, true)  <-> 0b000,
-  (true, true, true)   <-> 0b001,
-  (true, true, false)  <-> 0b010,
-  (true, false, false) <-> 0b011
+mapping encdec_mul_op : mul_op <-> bits(3) = {
+  struct { high = false, signed1 = true,  signed2 = true  } <-> 0b000,
```

Comment:
Maybe call these signed_rs1/signed_rs2?

Reply:
Can we resolve this conversation?  If so,  then I'll merge.

Reply:
This was not intended as a blocking comment, just a possible clarification. I'll let @Alasdair decide whether to make the change or not.